app/controllers/AuthenticationController.scala

```diff
@@ -429,6 +431,91 @@ class AuthenticationController @Inject()(
     }
   }
 
+  private lazy val oidcClientId = conf.WebKnossos.OIDC.clientId
+  private lazy val oidcProviderUrl = conf.WebKnossos.OIDC.providerURL
+
+  lazy val oidcConfig: OpenIdConnectConfig = OpenIdConnectConfig(oidcProviderUrl, oidcClientId)
+  lazy val absoluteOidcCallbackURL = "http://localhost:9000/api/auth/oidc/callback"
```

Comment:
Should be WkConf.Http.uri instead of hard-coded localhost:9000
```suggestion
  lazy val absoluteCallbackUrl = s"${conf.Http.uri}/api/auth/oidc/callback"
```

And I think it would be fair to move this to the OpenIdConnectClient class, as it is only used there

Reply:
I'm not sure about moving this to the oidcc class, because the client should exist independently of the controller and the callback url is very specific controller knowledge